generators/app/templates/ci/.circleci/config.yml

```diff
@@ -60,29 +60,29 @@ jobs:
       - store_artifacts:
           path: htmlcov
 
-  black:
+  format:
```

Comment:
Use same name for the job as for the make command (`format-check`, `lint-check`)